lib/main.dart

```diff
@@ -11,9 +14,23 @@ class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-        title: 'Flutter Demo',
-        theme: ThemeData(primarySwatch: Colors.blue, fontFamily: "OpenSans"),
-        home: const RootPage());
+    return MultiProvider(
+      providers: [
+         ChangeNotifierProvider<SignInProvider>(
+          create: (context) => SignInProvider(),
+        ),
+      ],
+      child: MaterialApp(
+          debugShowCheckedModeBanner: false,
+          title: 'Parking App',
+          theme: ThemeData(primarySwatch: Colors.blue, fontFamily: "OpenSans"),
+          initialRoute: "/SignIn",
+          routes: {
```

Comment:
I think you create file routes . Because after our code have routes and file main very maintain

Reply:
first screen is welcome... this is my test ... i want check my screen ... are you ok ?